doc/server.md

```diff
@@ -52,6 +53,7 @@ images.auto\_update\_interval       | integer   | global    | 6
 images.compression\_algorithm       | string    | global    | gzip                              | Compression algorithm to use for new images (bzip2, gzip, lzma, xz or none)
 images.default\_architecture        | string    | -         | -                                 | Default architecture which should be used in mixed architecture cluster
 images.remote\_cache\_expiry        | integer   | global    | 10                                | Number of days after which an unused cached remote image will be flushed
+instances.network.host\_name        | string    | global    | random                            | If it is set to `random` then use the random host interface names but if it's set to mac, then generate a name in the form `lxd<mac_address>`(MAC without leading 2 digits).
```

Comment:
@stgraber do you think we should use `instanes.nic.host_name` to indicate it controls the NIC host_name generation? Using "network" here seems a bit confusing.

Reply:
Sure, I'm fine with using `nic`

Reply:
Fixed.